genie-common/src/main/java/com/netflix/genie/common/util/Constants.java

```diff
@@ -0,0 +1,235 @@
+
+    /**
+     * File Path prefix to be used while creating paths for config files downloaded by Genie to local dir.
+     **/
+    public static final String CONFIG_FILE_PATH_PREFIX = "config";
+
+    /**
+     * Stderr Filename generated by Genie after running a job.
+     **/
+    public static final String STDERR_LOG_FILE_NAME = "stderr";
+
+    /**
+     * Stdout Filename generated by Genie after running a job.
+     **/
+    public static final String STDOUT_LOG_FILE_NAME = "stdout";
+
+    /**
+     * Done filename generated by Genie after running a job.
+     **/
+    public static final String GENIE_DONE_FILE_NAME = "./genie/genie.done";
+
+    /**
+     * Genie log file path.
+     **/
+    public static final String GENIE_LOG_PATH = "/genie/logs/genie.log";
+
+    /**
+     * File Path prefix to be used while creating directories for application files to local dir.
+     **/
+    public static final String APPLICATION_PATH_VAR = "applications";
+
+    /**
+     * File Path prefix to be used while creating directories for command files to local dir.
+     **/
+    public static final String COMMAND_PATH_VAR = "command";
+
+    /**
+     * File Path prefix to be used while creating directories for cluster files to local dir.
+     **/
+    public static final String CLUSTER_PATH_VAR = "cluster";
+
+    /**
+     * File Path prefix to be used while creating working directory for jobs.
+     **/
+    public static final String GENIE_PATH_VAR = "genie";
+
+    /**
+     * File Path prefix to be used while creating working directory for jobs.
+     **/
+    public static final String LOGS_PATH_VAR = "logs";
+
+    /**
+     * Environment variable for Genie working directory.
+     **/
+    public static final String GENIE_WORKING_DIR_ENV_VAR = "GENIE_WORKING_DIR";
+
+    /**
+     * Environment variable for Genie job working directory.
+     **/
+    public static final String GENIE_JOB_DIR_ENV_VAR = "GENIE_JOB_DIR";
+
+    /**
+     * Environment variable for Genie cluster directory.
+     **/
+    public static final String GENIE_CLUSTER_DIR_ENV_VAR = "GENIE_CLUSTER_DIR";
+
+    /**
+     * Environment variable for Genie command directory.
+     **/
+    public static final String GENIE_COMMAND_DIR_ENV_VAR = "GENIE_COMMAND_DIR";
+
+    /**
+     * Environment variable for Genie application directory.
+     **/
+    public static final String GENIE_APPLICATION_DIR_ENV_VAR = "GENIE_APPLICATION_DIR";
+
+    /**
+     * Process ID.
+     **/
+    public static final String PID = "pid";
+
+    /**
+     * Genie Done file contents prefix.
+     **/
+    public static final String GENIE_DONE_FILE_CONTENT_PREFIX = "printf '{\"exitCode\": \"%s\"}\\n' \"$?\" > ";
+
+    /**
+     * Protected constructor for utility class.
+     */
+    protected Constants() {
+    }
+
+    /**
+     * Enum containing types of files that the entities hold.
+     */
+    public enum FileType {
```

Comment:
I would move these enum's to their own class files so they're easier to find/modify.